pkg/scalers/liiklus/LiiklusService.pb.go

```diff
@@ -1,16 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.32.0
-// 	protoc        v4.23.2
```

Comment:
This file is unrelated to your changes, isn't it? These changes should not be here, I believe.

Reply:
Yes, the file is unrelated to my changes.